xknx/devices/sensor.py

```diff
@@ -16,7 +16,7 @@ class Sensor(Device):
     def __init__(self,
                  xknx,
                  name,
-                 group_address=None,
+                 group_address_state=None,
```

Comment:
Should we maybe keep `group_address` for a while and raise some sort of depreciation warning?

Reply:
I think this is fine.